executor/insert_test.go

```diff
@@ -581,435 +581,6 @@ func TestAllowInvalidDates(t *testing.T) {
 	runWithMode("ALLOW_INVALID_DATES")
 }
 
-func TestInsertWithAutoidSchema(t *testing.T) {
```

Comment:
Updated and moved to https://github.com/pingcap/tidb/pull/38808/files#diff-1c27a2956e8d617d4597580c00fb81d42bf81a4a7e726d911e11ac25ccb1e484R174